open-metadata-resources/open-metadata-deployment/charts/odpi-egeria-vdc/templates/apache-atlas.yaml

```diff
@@ -76,7 +76,7 @@ spec:
           image: "{{ if (.Values.image.configure.registry | default .Values.imageDefaults.registry) }}{{ .Values.image.configure.registry | default .Values.imageDefaults.registry }}/{{ end }}\
                   {{ if (.Values.image.configure.namespace | default .Values.imageDefaults.namespace) }}{{ .Values.image.configure.namespace | default .Values.imageDefaults.namespace }}/{{ end }}\
                   {{ .Values.image.configure.name }}\
-                  :{{ .Values.image.configure.tag | default .Values.imageDefaults.tag }}"
+                  :{{ .Values.image.egeria.tag | default .Values.egeria.version }}"
```

Comment:
This value is different from the others -- there were one or two other spots with this setup, too.  Here you have:

```
:{{ .Values.image.egeria.tag | default .Values.egeria.version }}
```

whereas elsewhere you have:

```
:{{ .Values.image.configure.tag | default .Values.egeria.version }}
```

I had assumed you meant to use the former everywhere (having a single tag for all Egeria images: both the Egeria core and the configure image), but perhaps it's these few places with the first setup that were a mistake (?)

Reply:
Thanks for clarifying. You are correct - I did have an error in that specific atlas file - but it seems the only one.

But I had intended to allow override on the two images independently, so the first example you list above is used for egeria, the second for configure. Both default to the egeria version

I've just added the additional commit

Reply:
Leave it to me to find the one that is wrong and assume it is correct and that all the others are incorrect 😝